aerogear/mobile-security-service-operator
# Upgrade operator sdk version from 0.8.1 to 0.9.0

## Motivation
Release of a new version of operator-SDK

## What
Upgrade the version of the framework

## Why
- Keep it easy to maintain and to be update

## Verification Steps
1. Check the CI
2. Install it locally and ensure that all was done with success by running `make install`

## Checklist:


## Progress


## NOTES
- It contains the changes performed at https://github.com/aerogear/mobile-security-service-operator/pull/158 and it blocked for it as well

IMAGE: docker.io/cmacedo/mobile-security-service-operator:UPGRADE_OPERATOR_SDK_VERSION
